src/OrbitGl/TimeGraph.h

```diff
@@ -127,7 +127,7 @@ class TimeGraph : public orbit_gl::CaptureViewElement, public orbit_gl::Timeline
   [[nodiscard]] orbit_gl::Batcher& GetBatcher() { return batcher_; }
 
   [[nodiscard]] const TimeGraphLayout& GetLayout() const { return layout_; }
-  [[nodiscard]] TimeGraphLayout& GetLayout() { return layout_; }
+  [[nodiscard]] orbit_gl::ImGuiTimeGraphLayout& GetImGuiLayout() { return layout_; }
```

Comment:
This function is only needed to get access to the `DrawProperties` member function in ImGuiTimeGraphLayout. It will be removed in one of the next PRs.